python/src/GaussianProcessFitterResult_doc.i.in

```diff
@@ -0,0 +1,194 @@
+%feature("docstring") OT::GaussianProcessFitterResult
+"Gaussian process fitter result.
+
+Parameters
+----------
+inputSample, outputSample : :class:`~openturns.Sample`
+    The samples :math:`(\vect{x}_k)_{1 \leq k \leq \sampleSize} \in \Rset^{\inputDim}` and :math:`(\vect{y}_k)_{1 \leq k \leq \sampleSize}\in \Rset^{\outputDim}`.
+metaModel : :class:`~openturns.Function`
+    The meta model: :math:`\tilde{\cM}: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim}`, defined in :eq:metaModelGPF.
+residuals : :class:`~openturns.Point`
+    The residual errors.
+relativeErrors : :class:`~openturns.Point`
+    The relative errors.
+basis :  :class:`~openturns.Basis`
+    Functional basis of size :math:`b` : :math:`(\varphi^l: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim})` for each :math:`l \in [1, b]`.
+    Its size should be equal to zero if the trend is not estimated.
+trendCoef : sequence of float
+    The trend coefficients vectors :math:`(\vect{\alpha}^1, \dots, \vect{\alpha}^{\outputDim})` stored as a Point.
+covarianceModel : :class:`~openturns.CovarianceModel`
+    Covariance function of the Gaussian process with its optimized parameters.
+optimalLogLikelihood : float
+    The maximum log-likelihood corresponding to the model.
+linAlgMethod : int
+    The used linear algebra method to fit the model.
+    0 for `LAPACK`, 1 for `HMAT`
```

Comment:
Isn't there any enum like GaussianProcessFitterResult.LinearAlgebra.LAPACK that we can access to set this variable, instead of a magic constant 0?

Reply:
The enum is defined at `GaussianProcessFitterResult` level. Thus 

```python
import openturns as ot
import openturns.experimental as otexp
print(otexp.GaussianProcessFitterResult.LAPACK)
```
returns `0`.

Reply:
Ok, good. Therefore the doc should reflect this, so that the user can know about it.